### PR TITLE
Expanded FAQ

### PR DESCRIPTION
**Pull request by nicoongit**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -28,6 +40,17 @@ To change your password, sign in to the [Developers portal](https://developers.a
 
 If you haven't received a confirmation mail, it is often because the email address was entered incorrectly. Please sign in to the [Developers portal](https://developers.amadeus.com/){:target="\_blank"} and visit the **My Account** section to confirm that the email address used to create the account is correct. If so, please check your spam folder for an email from noreply@amadeus.com.
 
+## Business enquiries
+
+### How can I monetise my application?
+
+You are free to create your own business models around our APIs, such as charging users to use our APIs in their apps or adopting a subscription-based model. However, we do not give any commission or other types of incentives for the Self-Service API. The latter is only possible in the Enterprise framework.
```
Comment (Matpinc) [inline docs/faq.md:47]: In the case of flight booking, users can also monetise their apps by adding a markup on flight offers.

File: docs/faq.md [hunk 2]
```diff
@@ -204,6 +336,99 @@ You can consult booking made through Flight Create Orders using the Flight Order
...
+
+The maximum date for the 'checkInDate' parameter is 359 days from today. Anything beyond this will return the error message 'MAXIMUM ADVANCE DAYS BOOKING EXCEEDED'.
+
+### There is a mismatch in the amenities returned
+
+The amenities shown as examples in the API reference documentation may not match some of the results returned. The list of amenities in our documentation is the ones that we are able to index with our API. The hotel content is fed by the hotel providers, and this is not standard, as each hotel is free to add whatever content they want to the amenities list.
+
+### How to search a hotel by location
+
+Regarding the input for a specific location in a hotel search, you have the following options:
```
Comment (Matpinc) [inline docs/faq.md:387]: We can no longer search by IATA codes with Hotel Search v3. In order to search by location you will need to use the third endpoint of Hotel List (/reference-data/locations/hotels/by-geocode), which allows you to search using a latitude & longitude, and returns hotelIds from the specific search coordinates. You will then need to use this hotelId in the third endpoint of the Hotel Search API.

File: docs/faq.md [hunk 3]
```diff
@@ -204,6 +336,99 @@ You can consult booking made through Flight Create Orders using the Flight Order
...
+
+As of now, the hotel booking API does not allow canceling rooms. If this option is possible with your hotel offer, the cancellation will have to be done manually with the hotels. We are working on an API for the cancellation; however, it is still too soon to commit to anything.
+
+### Why do I get 500 status code?
+
+The process of booking a hotel in the test environment involves sending your request to each hotel provider, and each provider has its own environment and rules. Due to these differences, there may be connectivity issues with the providers that can result in a timeout. Additionally, if many requests are sent to a particular hotel, they may choose to block them. If you provide us with a timestamp and details of another API request that has failed, including the hotel in question, we can search our logs to find more information. However, it is likely that the issue is one of the aforementioned cases.
+
+### How can I see Amadeus API coverage for a hotel chain?
+
+You can use the Hotel Search API to search the hotels by their hotel Id.
```
Comment (Matpinc) [inline docs/faq.md:418]: You can find the list of supported hotel chains in our data collection: https://github.com/amadeus4dev/data-collection/blob/master/data/hotelchains.md.

File: docs/faq.md [hunk 4]
```diff
@@ -204,6 +336,99 @@ You can consult booking made through Flight Create Orders using the Flight Order
...
+
+You can use the Hotel Search API to search the hotels by their hotel Id.  
+
+### What is considered a query?
+
+When you make an initial call to the Hotel Search API with a cityCode parameter, it is considered one query that returns 10 hotels. If you then click on each hotel to view room details and other information, you are making additional API calls using the second endpoint of the Hotel Search. Each hotel that you click on generates another API call. For example, if you click on 5 hotels to see room details, you will be making a total of 5 additional queries.
+
+### Do I need any legal documents to make a booking?
+
+No, there are no legal documents required.
```
Comment (Matpinc) [inline docs/faq.md:426]: You will need to comply with any local legal requirements in your market.

File: docs/faq.md [hunk 5]
```diff
@@ -223,6 +448,11 @@ Different airline consolidators handle payments in different ways. In some cases
 
 For Self-Service users, all post-booking services must be handled offline with the consolidator you work with for ticket issuance. In general, these actions can be made while the Passenger Name Record (PNR) is queued for ticketing (before the ticket is issued), though their availability once a ticket has issued depends largely on the consolidator and the clauses of your agreement. 
 
+
+### How can I get a consolidator?
+
+Before requesting a consolidator, please first make sure that you are approved markets for Flight Create Orders. You need it to implement flight booking in Self-Service. Once this is verified, please go to the Support section and get in touch with us using the Contact form.
```
Comment (Matpinc) [inline docs/faq.md:454]: Before requesting a consolidator, please first make sure that you are **in one of the** approved markets for Flight Create Orders. You need it to implement flight booking in Self-Service. Once this is verified, please go to the Support section and get in touch with us using the Contact form.

File: docs/faq.md [hunk 6]
```diff
@@ -5,15 +5,27 @@ This page provides help with the most common questions about Amadeus Self-servic
...
...-self-service-apis?)</li><li>[How do I access the Self-Service APIs documentation?](#how-do-i-access-the-self-service-apis-documentation?)</li><li>[Do you provide SDKs?](#do-you-provide-sdks)</li><li>[Where can I see code examples for Amadeus Self-Service APIs?](#where-can-i-see-code-examples-for-amadeus-self-service-apis)</li><li>[How do I make my first Self-Service API call?](#how-do-i-make-my-first-self-service-api-call)</li><li>[How do I move Self-Service APIs from test to production?](#how-do-i-move-self-service-apis-from-test-to-production)</li><li>[How do I delete my application built using Self-Service APIs?](#how-do-i-delete-my-application-built-using-self-service-apis)</li><li>[Will you include more APIs in the Self-Service catalog?](#will-you-include-more-apis-in-the-self-service-catalog?)</li><li>[What are the terms of service for Amadeus Self-Service APIs?](#what-are-the-terms-of-service-for-amadeus-self-service-apis)</li><li>[I am not a travel agent and have no experience in the travel industry, can I still use the Self-Service APIs?](#i-am-not-a-travel-agent-and-have-no-experience-in-the-travel-industry,-can-i-still-use-the-self-service-apis)</li></ul>              |
```
Comment (minjikarin) [inline docs/faq.md:9]: I see that there are several individual APIs sections below, shall we change this section to be 'Self-Service APIs 
General' or 'General Self-Service APIs'? I was looking for some API functional questions here but it is not.

Comment (tsolakoua) [reply docs/faq.md:9]: I agree with @minjikarin Nico, a name like "general 'as Minji suggested, or "overview" or "framework" could make it clear

Comment (nicoongit) [reply docs/faq.md:9]: Thanks @minjikarin , @tsolakoua , changed to **Self-Service APIs general**

**Review by tsolakoua — APPROVED** [review]

**Pull request merged**